### ISSUE TITLE
Bam CoralineAda for violating the Code Of Conduct

### ISSUE THREAD

**Issue by SilicaAndPina** [Microsoft/MS-DOS#45]
Tfw you cant follow your own Code Of Conduct:

https://twitter.com/CoralineAda/status/1029162264979546112

**Comment by henrycjc** [comment]
Don't you have something better to do you weeb freak? 

...says the guy also posting in the thread

**Comment by aphextwins** [comment]
they wont cause "her" privilege level outranks cis white males. so CoC doesnt apply to her.

**Comment by acao** [comment]
why is this relevant? her anger seems righteous. does code of conduct apply to private life?

**Comment by acao** [comment]
yes i did mr richard d james

**Comment by Themesaul** [comment]
I hope that one day there will be respect for all. Without warriors that go over the line. #Ban

**Comment by acao** [comment]
@aphextwins yes my bad, not used to hearing this kind of rhetoric from outside my demographic, but just wondering - whats with the double quotes around "her"?

**Comment by PixleProgrambetaxl** [comment]
+1

**Comment by aphextwins** [comment]
@acao Because CoralineAda has a penis and XY chromosomes. so he is a he, and not a she.
